pkg/ddc/alluxio/ufs.go

```diff
@@ -106,42 +91,38 @@ func (e *AlluxioEngine) UpdateUFS(updatedUFSMap map[string][]string) (err error)
 		e.Log.Error(err, "Failed to add or remove mount points")
 		return err
 	}
+	return nil
+}
 
-	//3. update dataset status to updated
-	err = e.SetUFSUpdated()
+func (e *AlluxioEngine) UpdateOnUFSChange() (updateReady bool, err error) {
+	// 1. get the dataset
+	dataset, err := utils.GetDataset(e.Client, e.name, e.namespace)
 	if err != nil {
-		e.Log.Error(err, "Failed to update dataset status to updated")
-		return err
+		e.Log.Error(err, "Failed to get the dataset")
+		return
 	}
 
-	return err
-}
-
-func (e *AlluxioEngine) UpdateOnUFSChange() (updateReady bool, err error) {
-	// 1.get the updated ufs map
-	// updatedUFSMap, err
-	updatedUFSMap, err := e.GetUpdateUFSMap()
+	// 2. if no need to add/remove mount points, return false immediately
+	if len(dataset.Spec.Mounts) == len(dataset.Status.Mounts) {
```

Comment:
I think we just use `updateUFSMap` to check if the mount points are changed.

Reply:
@cheyang @TrafalgarZZZ It's just used to quickly exit.